rootex/core/resource_loader.cpp

```diff
@@ -169,30 +172,50 @@ VisualModelResourceFile* ResourceLoader::CreateVisualModelResourceFile(const Str
 		return nullptr;
 	}
 
+	const aiScene* scene = s_ModelLoader.ReadFile(path,
+	      aiProcess_CalcTangentSpace      |
+		  aiProcess_Triangulate           | 
+		  aiProcess_JoinIdenticalVertices | 
+		  aiProcess_SortByPType);
+    const aiMesh* mesh = scene->mMeshes[0]; //Taking single mesh
+	aiFace* face;
+
 	// File not found in cache, load it only once
-	PANIC(s_ModelLoader.LoadFile(OS::GetAbsolutePath(path).generic_string()) == false, "Model could not be loaded: " + OS::GetAbsolutePath(path).generic_string());
+	PANIC(scene == false, "Model could not be loaded: " + OS::GetAbsolutePath(path).generic_string());
```

Comment:
Compare with nullptr instead of false. It is unidiomatic to compare pointers with bool directly